### PR TITLE
Start of component upgrades.

### PR DESCRIPTION
**Pull request by bweston92**
I've created a branch on this repo for you to easily contribute.

- Firstly, I've upgrades the components to the versions we have mentioned in #17.
- Secondly, I've decided to use the existing PDO connection by default, meaning less configuration. Will revert at your discretion.
- I've temporarily remove the snapshot factories until I understand the changes.
- Note, this as it stands doesn't hold configuration updates.
- Switched to PDO event store using MySQL by default as Doctrine adapter is deprecated.

### CODE REVIEW

File: config/dependencies.php [hunk 1]
```diff
@@ -13,16 +13,11 @@
 return [
     // prooph/event-store set up
     \Prooph\EventSourcing\EventStoreIntegration\AggregateTranslator::class => InvokableFactory::class,
-    \Prooph\EventStore\EventStore::class => \Prooph\EventStore\Container\EventStoreFactory::class,
-    \Prooph\EventStore\Adapter\Doctrine\DoctrineEventStoreAdapter::class => \Prooph\EventStore\Adapter\Doctrine\Container\DoctrineEventStoreAdapterFactory::class,
-    // prooph snapshot setup
-    \Prooph\Snapshotter\SnapshotPlugin::class => \Prooph\Snapshotter\Container\SnapshotPluginFactory::class,
```
Comment (bweston92) [inline config/dependencies.php:21]: @prolic can you guide on how to set Snapshots up with latest changes.

**Comment by bweston92** [comment]
I can confirm that you can at least get a `composer install` working with this package now where as I couldn't before on Laravel 5.3.

File: config/dependencies.php (hunk 1 above)
Comment (prolic) [reply config/dependencies.php:21]: https://github.com/prooph/proophessor-do/blob/master/config/autoload/mongo_snapshot_store.local.php.dist

https://github.com/prooph/proophessor-do/blob/master/config/autoload/pdo_snapshot_store.local.php.dist

**Review by codeliner — APPROVED** [review]

**Comment by codeliner** [comment]
looks good, thx for the work @bweston92

**Pull request merged**